sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -156,6 +156,9 @@ struct _pi_device : _pi_object {
   // Level Zero device handle.
   ze_device_handle_t ZeDevice;
 
+  // Keep the subdevices that are partitioned from this pi_device for reuse
+  std::vector<pi_device> SubDevices;
```

Comment:
Please add a note that the order is significant and repeats the order in which L0 returns handles

Reply:
Thanks. Done.